applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -100,7 +102,20 @@ <h3 class="signup-confirmation--success">Check your Inbox!</h3>
 <div class="l-side-margins">
 <div class="content">
     <div class="gs-container">
-        <div class="newsletters-heading">Guardian newsletters: sign up</div>
+        <div class="newsletters-heading__container">
+            <div class="newsletters-heading">
+                Guardian newsletters: sign&nbsp;up
+            </div>
+            <div class="newsletters-heading__manage-button-wrapper">
+                <a
+                data-link-name="all-newsletters : manage-emails"
+                href=@manageEmailsUrl>
```

Comment:
I think you might be missing double quotes around the value here?

Reply:
Good catch. I thought it was coming through from the string declared as a variable, but I think actually the browser was adding in the double quotes automatically. Good chance this would have broken on some random browser at some point so thanks for spotting it